### PR TITLE
added support for https odoo instance

### PR DESCRIPTION
**Pull request by manjeet-bhatkar**

### CODE REVIEW

File: lib/odoo.js [hunk 1]
```diff
@@ -60,4 +63,4 @@ class Odoo {
 	}
 }
 
-module.exports = Odoo;
+module.exports = Odoo;
```
Comment (SamVerschueren) [inline lib/odoo.js]: Add final newline

File: lib/request.js [hunk 2]
```diff
@@ -17,7 +17,9 @@ module.exports = (connection, path, params) => {
...
 			Cookie: `${connection.sid};`
 		}
-	});
+	};
+
+	const client = connection.protocol === 'https' ? jayson.client.https(config) : jayson.client.http(config);
 
 	return pify(client.request.bind(client))('call', params || {});
-};
+};
```
Comment (SamVerschueren) [inline lib/request.js]: Add final newline

**Review by SamVerschueren — CHANGES_REQUESTED** [review]
Can you also rebase against master? Did a minor change.

**Comment by manjeet-bhatkar** [comment]
Hi @SamVerschueren , I have made changes and rebase against master. Please check this PR
https://github.com/manjeet-bhatkar/odoo-connect/pull/2

**Comment by ivantodorovich** [comment]
Same problem here. What does it take for this to get merged?

**Review by SamVerschueren — APPROVED** [review]

**Comment by SamVerschueren** [comment]
Thanks for the PR! Will release a new version soon.

**Pull request merged**